pkg/client/cli/telepresence_test.go

```diff
@@ -413,7 +413,7 @@ func (ts *telepresenceSuite) TestB_Connected() {
 }
 
 func (ts *telepresenceSuite) TestC_Uninstall() {
-	//telepresence(ts.T(), "connect")
+	// telepresence(ts.T(), "connect")
```

Comment:
if this line isn't gonna be used could you actually delete it instead?

Reply:
uncommenting this line allows this test to be run as a standalone

Reply:
ah okay fair enough, could you add a  comment to that effect above it? otherwise it might get deleted as dead code